charts/k8spacket/values.yaml

```diff
@@ -87,4 +87,9 @@ k8sPacket:
         ttl: "24h"
     metrics:
       ## Enabled/disabled exposing TLS Prometheus metrics
-      enabled: true
\ No newline at end of file
+      enabled: true
+  volumes:
+    ## path to kernel tracing folder on instance
+    ## default to /sys/kernel/tracing
+    ## on AmazonLinux this should be set to /sys/kernel/debug/tracing
```

Comment:
`AmazonLinux` is not enough here because on Amazon Linux 2023 the path `/sys/kernel/tracing` is also valid.
```
ssh …
   ,     #_
   ~\_  ####_        Amazon Linux 2023
  ~~  \_#####\
  ~~     \###|
  ~~       \#/ ___   https://aws.amazon.com/linux/amazon-linux-2023
   ~~       V~' '->
    ~~~         /
      ~~._.   _/
         _/ _/
       _/m/'
$ mount -t tracefs
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime,seclabel)
tracefs on /sys/kernel/debug/tracing type tracefs (rw,nosuid,nodev,noexec,relatime,seclabel)
```

Let's set this comment to:
```
    ## path to kernel tracing folder on an instance
    ## default to '/sys/kernel/tracing'
    ## check by command: 'mount -t tracefs'
    ## f.e. on 'Amazon Linux 2 Kernel 5.10' this should be set to '/sys/kernel/debug/tracing'

Reply:
Good point. I updated the value docs and increased the chart version as well.